var/spack/repos/builtin/packages/py-galaxy-util/package.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyGalaxyUtil(PythonPackage):
+    """Galaxy Generic Utilities"""
+
+    homepage = "https://github.com/galaxyproject/galaxy"
+    pypi = "galaxy-util/galaxy-util-22.1.2.tar.gz"
+
+    version("22.1.2", sha256="80257c94dc9122ebf80d643aa3962fe8beda23dbba8fc4820a0d2b720f479f98")
+
+    depends_on("python@3.7:3.10", type=("build", "run"))
+    depends_on("py-poetry", type="build")
+
+    depends_on("py-a2wsgi", type=("build", "run"))
```

Comment:
Same comments here, 22.1.2 does not restrict Python, uses setuptools instead of poetry, and only has the following deps:
```
bleach
boltons
docutils
importlib_resources
MarkupSafe
packaging<22  # packaging 22.0 dropped LegacyVersion
pycryptodome
pyparsing
PyYAML
requests
Routes
typing-extensions
zipstream-new
```

Reply:
done